packages/components/package.json

```diff
@@ -22,6 +22,7 @@
     "lint": "eslint -f codeframe \"./**/*.{js,jsx}\"",
     "build": "rollup --config rollup.config.js",
     "start": "start-storybook -p 3000 --ci",
+    "start:ssr-preview": "SSR=true start-storybook -p 3000 --ci --preview-url=/ssr-iframe",
```

Comment:
suggestion on how to extract this part? 

maybe `node ../../presets/react/scripts/start-storybook-ssr.js` ?